scripts/test_platform.sh

```
@@ -0,0 +1,178 @@
+echo -e "  ${BLUE}Test Server:${NC}   tail -f logs/test-server.log"
+
+# Create PID file for easy cleanup
+echo "$STREAM_ENGINE_PID" > .stream-engine.pid
+echo "$TEST_SERVER_PID" > .test-server.pid
+
+# Set up trap to clean up on exit
+cleanup() {
+    echo -e "\n${YELLOW}Shutting down services...${NC}"
+    
+    if [ -f .stream-engine.pid ]; then
+        kill $(cat .stream-engine.pid) 2>/dev/null || true
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)